storage/remote/generic/generic.go

```diff
@@ -62,16 +69,15 @@ func (c *Client) Store(samples model.Samples) error {
 		}
 		req.Timeseries = append(req.Timeseries, ts)
 	}
-	client := NewGenericWriteClient(c.conn)
 	ctxt, _ := context.WithTimeout(context.Background(), c.timeout)
```

Comment:
Ok cool, good to know. I don't have a lot of experience with context best practices yet. So I'll add that cancel.